spark/src/test/java/org/apache/zeppelin/spark/PySparkInterpreterTest.java

```diff
@@ -123,6 +123,21 @@ public void testCompletion() {
     }
   }
 
+  @Test
+  public void testRedefinitionZeppelinContext() {
+    if (getSparkVersionNumber() > 11) {
+      String redefinitionCode = "z = 1\n";
```

Comment:
Hello.
If you do not mind, can you tell me more details?
I think I did the verification procedure properly.
Do you have any problems?

```
//  "z = 1\n";
assertEquals(InterpreterResult.Code.SUCCESS, pySparkInterpreter.interpret(redefinitionCode, context).code()); 
// "z.input(\"test\")\n";  <-- If z is reassign, an error occurs.
assertEquals(InterpreterResult.Code.ERROR, pySparkInterpreter.interpret(validCode, context).code());
```